.circleci/config.yml

```diff
@@ -7,24 +7,30 @@ executors:
   node_executor:
     docker:
       - image: circleci/node:12.9.0-stretch
+  ruby_executor:
+    docker:
+      - image: circleci/ruby:2-stretch
+
```

Comment:
Would recommend adding this line to both exectuors
```    working_directory: ~/project
```
And you can remove the './' for the checksum; either way works though :)

Reply:
for the ./ it was to be consistant among all the file paths as required by @EdJoJob on the EthSigner PR similar to this one.